vulture_os/toolkit/api_parser/sentinel_one_mobile/sentinel_one_mobile.py

```diff
@@ -201,5 +203,3 @@ def execute(self):
 
         logger.info(f"[{__parser__}]:execute: update last_api_call to {self.frontend.last_api_call}",
                     extra={'frontend': str(self.frontend)})
```

Comment:
Try to keep the same logging as in the other parsers
```suggestion
                logger.info(f"[{__parser__}]:execute: Parsing done.", extra={'frontend': str(self.frontend)})
```